src/Database/LSMTree/Internal.hs

```diff
@@ -1347,6 +1345,10 @@ data BloomFilterAlloc =
 -- persistence of the config to disk.
 deriving stock instance Read BloomFilterAlloc
 
+bloomFilterAllocForLevel :: BloomFilterAlloc -> LevelNo -> RunBloomFilterAlloc
+bloomFilterAllocForLevel (AllocFixed n) _        = RunAllocFixed n
+bloomFilterAllocForLevel (AllocRequestFPR fpr) _ = RunAllocRequestFPR fpr
```

Comment:
Ah, so this is why there are two types. Once we add support for Monkey calculations, there will be a third option for `BloomFilterAlloc`, but that will translate to one of the existing constructors in `RunBloomFilterAlloc`, right?

Reply:
I might add a separate constructor to `RunBloomFilterAlloc`, but yes the types `BloomFilterAlloc` and `RunBloomFilterAlloc` would be slightly different